recipe/conda_build_config.yaml

```diff
@@ -70,27 +72,29 @@ flann:
 fontconfig:
   - 2.12
 freetype:
-  - 2.8                # [not ppc64le]
+  - 2.8.1              # [not ppc64le]
 gstreamer:
   - 1.12
 gst_plugins_base:
   - 1.12
```

Comment:
These were missing from `pin_the_slow_way.py`. Though it is good to see them here.